lldb/source/API/SBThread.cpp

```diff
@@ -774,7 +798,10 @@ SBError SBThread::StepOverUntil(lldb::SBFrame &sb_frame,
   char path[PATH_MAX];
 
   std::unique_lock<std::recursive_mutex> lock;
-  ExecutionContext exe_ctx(m_opaque_sp.get(), lock);
+  Process::StopLocker stop_locker;
+  ExecutionContext exe_ctx(m_opaque_sp.get(), lock, stop_locker);
+  if (!stop_locker.IsLocked())
+    return sb_error;
```

Comment:
We shouldn't return an empty error when we know the error...  Again, if the ExecutionContext constructor took an optional error, that would be simple to do.